defines/defines.go

```diff
@@ -35,6 +35,7 @@ const (
 	BatchIntervalUploadStats        = 0.95
 	ChannelInBufferSize             = 800
 	ChannelOutBufferSize            = 800
+	DefaultConfigFile               = "/etc/arachne/arachne.yaml"
```

Comment:
Not sure what you mean, @Vasilis  - the CLI defaults to what we define here if --config_file is not passed. Are you suggesting we remove this line and hard code code it in the ParseCliArgs function?